src/sso/sso.ts

```diff
@@ -9,6 +8,21 @@ import { ListConnectionsOptions } from './interfaces/list-connections-options.in
 import { ProfileAndToken } from './interfaces/profile-and-token.interface';
 import { Profile } from './interfaces/profile.interface';
 
+const toQueryString = (options: Record<string, string | undefined>): string => {
+  const searchParams = new URLSearchParams();
+  const keys = Object.keys(options).sort();
```

Comment:
`query-string` was also sorting the params under the hood. This at least keeps the output deterministic, which is a good thing for the existing test suite.